lib/interface_quda.cpp

```diff
@@ -108,6 +108,30 @@ CloverField *cloverEigensolver = nullptr;
 GaugeField momResident;
 GaugeField *extendedGaugeResident = nullptr;
 
+/** 
+ * callMultiSrcQuda related gauge split
+ * update_split_gauge : 
+ *   QUDA_UPDATE_SPLITE_GAUGE_OFF delete buffer after usage
+ *   QUDA_UPDATE_SPLITE_GAUGE_FALSE split gauge in buf
```

Comment:
"in buf" probably means reusing the split gauge that has been backed up previously?